sql/updates/latest-dev.sql

```diff
@@ -55,3 +55,21 @@ ALTER FUNCTION _timescaledb_internal.rxid_out(@extschema@.rxid) SET SCHEMA _time
 
 ALTER TABLE _timescaledb_config.bgw_job
     ALTER COLUMN owner SET DEFAULT pg_catalog.quote_ident(current_role)::regrole;
+
+-- Log with events that will be sent out with the telemetry. The log
+-- will be flushed after it has been sent out.
+CREATE TABLE _timescaledb_catalog.telemetry_event (
+       created timestamptz NOT NULL DEFAULT current_timestamp,
+       tag name NOT NULL,
+       body jsonb NOT NULL
+);
+
+SELECT pg_catalog.pg_extension_config_dump('_timescaledb_catalog.telemetry_event', '');
+
+GRANT SELECT ON _timescaledb_catalog.telemetry_event TO PUBLIC;
+
+DROP FUNCTION IF EXISTS @extschema@.get_telemetry_report();
+
+CREATE FUNCTION @extschema@.get_telemetry_report(truncate_telemetry_events bool = false)
```

Comment:
is "wiping clean a table" is something which should happen in a `get` function?

Reply:
This function declaration seem to be part of the public api - If its used only in testing; it should be declared only during that...we are also maintaining it via drop/creates between versions - if thats not warranted ; why keep it?

I wonder if there are any obstacles which prevents the tests do something like:
```
truncate _timescaledb_catalog.telemetry_event;
```

Reply:
> This function declaration seem to be part of the public api - If its used only in testing; it should be declared only during that...we are also maintaining it via drop/creates between versions - if thats not warranted ; why keep it?

Hmm... good point.

Reply:
Removed the changes to the `get_telemetry_report` function.